internal-api/src/main/kotlin/ua/anastasiia/finesapp/NatsSubject.kt

```diff
@@ -0,0 +1,49 @@
+package ua.anastasiia.finesapp
+
+object NatsSubject {
+    private const val INPUT_PORT = "input"
+    private const val OUTPUT_PORT = "output"
+    private const val REQREPLY_COMMUNICATION_TYPE = "reqreply"
+    private const val PUBSUB_COMMUNICATION_TYPE = "pubsub"
+
+    open class Subject(private val subdomain: String) {
+        protected fun inputReqReply(action: String) = "$INPUT_PORT.$REQREPLY_COMMUNICATION_TYPE.$subdomain.$action"
+        protected fun outputPubSub(action: String) = "$OUTPUT_PORT.$PUBSUB_COMMUNICATION_TYPE.$subdomain.$action"
```

Comment:
from usages I see, that the action parameter is not an action but an entity identifier

Reply:
I can agree that here, the parameter is not action but event:
`protected fun outputPubSub(event: String) = "output.pubsub.$subdomain.$event"`

but what about usages of this:
```
protected fun inputReqReply(action: String) = "input.reqreply.$subdomain.$action"
val ADD = inputReqReply("add")
```
like add is an action, isn't it?